build.gradle

```diff
@@ -25,7 +25,7 @@ tasks.withType(JavaCompile) {
 jib {
   container.ports = ['8081']
   container.args = []
-  from.image = 'quay.io/sdase/openjdk-runtime:11-hotspot-distroless'
+  from.image = 'gcr.io/distroless/java11-debian11:nonroot'
```

Comment:
* I'd make this a "feat" commit
* Are there any reasons to not migrate to Java 17?

Reply:
> I'd make this a "feat" commit
Done.

> Are there any reasons to not migrate to Java 17?
One step after the other. I tried using Java 17 as a fix for the failed [Jib 3.2.2 upgrade](https://github.com/SDA-SE/mongodb-operator/pull/272) but it did not build right away. Now we have a new Gradle version and other upgrades, so we could try again.

Reply:
> I'd make this a "feat" commit

Done.

> Are there any reasons to not migrate to Java 17?

One step after the other. I tried using Java 17 as a fix for the failed [Jib 3.2.2 upgrade](https://github.com/SDA-SE/mongodb-operator/pull/272) but it did not build right away. Now we have a new Gradle version and other upgrades, so we could try again.